src/js/components/FocusedContainer.js

```diff
@@ -18,44 +13,51 @@ export const FocusedContainer = ({
   const [bodyOverflowStyle, setBodyOverflowStyle] = useState('');
   const ref = useRef(null);
 
+  const root = useContext(RootsContext);
+  const [updatedRoot, setUpdatedRoot] = useState(root);
+
   useEffect(() => {
-    const removeTrap = () => {
-      const child = ref.current;
-      getBodyChildElements()
-        .filter(isNotAncestorOf(child))
-        .forEach(makeNodeFocusable);
-      if (restrictScroll) {
-        document.body.style.overflow = bodyOverflowStyle;
-      }
-    };
+    // make sure value of null is not added to array
+    if (ref.current) setUpdatedRoot([...root, ref.current]);
+  }, [root]);
 
-    const handleTrapFocus = () => {
-      const child = ref.current;
-      getBodyChildElements()
-        .filter(isNotAncestorOf(child))
-        .forEach(makeNodeUnfocusable);
+  useEffect(() => {
+    if (
+      bodyOverflowStyle !== 'hidden' &&
+      !hidden &&
+      restrictScroll &&
+      trapFocus
```

Comment:
do we need `trapFocus` here?

Reply:
I believe so, because if we're not trapping focus in the drop we would want the body to still be able to scroll in case an element on the page is receiving focus

Reply:
However, after thinking through it some more, it seems that "restrictScroll" should always be `true` when we want this to happen. The reason I had included it though was because it is part of the existing logic, but doing some testing I think we may be able to take it out.